Example/PaymentSheet Example/PaymentSheetUITest/EmbeddedUITest.swift

```diff
@@ -293,6 +368,110 @@ class EmbeddedUITests: PaymentSheetUITestCase {
         XCTAssertTrue(applePayButton.waitForExistence(timeout: 3.0))
         XCTAssertTrue(applePayButton.isSelected)
     }
+    
+    func testSelection() {
+        var settings = PaymentSheetTestPlaygroundSettings.defaultValues()
+        settings.mode = .paymentWithSetup
+        settings.uiStyle = .paymentSheet
+        settings.customerKeyType = .legacy
+        settings.formSheetAction = .continue
+        settings.customerMode = .new
+        loadPlayground(app, settings)
+        
+        // Start by saving a new card
+        app.buttons["Present PaymentSheet"].waitForExistenceAndTap()
+        try! fillCardData(app, postalEnabled: true)
```

Comment:
I think we can use a .returning customer here so we don't have to fillCardData?

Reply:
Yeah I'd like to, but the returning customer has two PMs saved and the one that is selected by default changes randomly between a card and bank making it not great for tests.